code-postprocessing/bbob_pproc/ppfigdim.py

```diff
@@ -480,6 +483,13 @@ def plot(dsList, valuesOfInterest=None, styles=styles):
             res.extend(plt.plot(tmp[:, 0], maxevals / tmp[:, 0]**ynormalize_by_dimension,
                        color=styles[len(valuesOfInterest) - 1]['color'],
                        ls='', marker='x', markersize=20))
+            if no_target_reached:
+                ylim_maxevals = max(maxevals / tmp[:, 0]**ynormalize_by_dimension)
+                ylim = (ylim[0], math.ceil(ylim_maxevals))
+                plt.annotate('no target reached', xy=(0.5, 0.5), xycoords='axes fraction', fontsize=14,
+                            horizontalalignment='center', verticalalignment='center')
+            else:
+                ylim = (ylim[0], math.ceil(ylim[1]))
```

Comment:
I have tried to change the code as you suggested, that is without using `math.ceil`:
```
if 1 < 3: # maxevals
            ylim = plt.ylim()
            res.extend(plt.plot(tmp[:, 0], maxevals / tmp[:, 0]**ynormalize_by_dimension,
                       color=styles[len(valuesOfInterest) - 1]['color'],
                       ls='', marker='x', markersize=20))
            if no_target_reached:
                ylim_maxevals = max(maxevals / tmp[:, 0]**ynormalize_by_dimension)
                ylim = (ylim[0], ylim_maxevals)
                plt.annotate('no target reached', xy=(0.5, 0.5), xycoords='axes fraction', fontsize=14,
                            horizontalalignment='center', verticalalignment='center')
            plt.ylim(ylim)
```
However, the obtained result is not the same, for example : 
- With the current code: 
[ppfigdim_f007.pdf](https://github.com/numbbo/coco/files/467257/ppfigdim_f007.pdf)
[ppfigdim_f023.pdf](https://github.com/numbbo/coco/files/467258/ppfigdim_f023.pdf)
- With the revised code (without using `math.ceil`):  
[ppfigdim_f007.pdf](https://github.com/numbbo/coco/files/467260/ppfigdim_f007.pdf)
[ppfigdim_f023.pdf](https://github.com/numbbo/coco/files/467262/ppfigdim_f023.pdf)